cypress/e2e/item/view/viewFolder.cy.ts

```diff
@@ -89,11 +89,25 @@ describe('View Folder', () => {
 
   it('search', () => {
     const { id } = parentItem;
+    const searchText = child1.name;
     cy.visit(buildItemPath(id, { mode: ItemLayoutMode.Grid }));
 
     cy.get(`#${buildItemCard(child1.id)}`).should('be.visible');
 
-    cy.get(`#${ITEM_SEARCH_INPUT_ID}`).type(child1.name);
+    cy.wait('@getChildren');
+    cy.wait('@getChildren');
+
+    cy.get(`#${ITEM_SEARCH_INPUT_ID}`).type(searchText);
+
+    cy.wait('@getChildren').then(({ request: { query } }) => {
```

Comment:
It is possible to type the query to avoid the `as unknown` ? If not it's okay like that ^^